media-libs/amdgpu-pro-vulkan/amdgpu-pro-vulkan-21.40.2.1350682.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit unpacker
+
+MY_PV_HIGH=$(ver_cut 1)
+MY_PV_MIDDLE=$(ver_cut 2)
+MY_PV_LOW=$(ver_cut 3)
+MY_PV_REV=$(ver_cut 4)
+
+MY_PV="${MY_PV_HIGH}.${MY_PV_MIDDLE}.${MY_PV_LOW}"
+MY_PV_FULL="${MY_PV}-${MY_PV_REV}"
+
+DESCRIPTION="AMD's closed source vulkan driver, from Radeon Software for Linux"
+HOMEPAGE="https://www.amd.com/en/support"
+SRC_URI="
+abi_x86_64? ( https://repo.radeon.com/amdgpu/${MY_PV}/ubuntu/pool/proprietary/v/vulkan-amdgpu-pro/vulkan-amdgpu-pro_${MY_PV_FULL}_amd64.deb -> vulkan-amdgpu-pro_${MY_PV}_amd64.deb )
+abi_x86_32? ( https://repo.radeon.com/amdgpu/${MY_PV}/ubuntu/pool/proprietary/v/vulkan-amdgpu-pro/vulkan-amdgpu-pro_${MY_PV_FULL}_i386.deb -> vulkan-amdgpu-pro_${MY_PV}_i386.deb )
```

Comment:
I think it could be a good thing to make a MY_LINK to shorten these ones, like  

`MY_LINK="https://repo.radeon.com/amdgpu/${MY_PV}/ubuntu/pool/proprietary/v/vulkan-amdgpu-pro"  `

and  

```
SRC_URI="
abi_x86_64? ( ${MY_LINK}/vulkan-amdgpu-pro_${MY_PV_FULL}_amd64.deb -> vulkan-amdgpu-pro_${MY_PV}_amd64.deb )
abi_x86_32? (${MY_LINK}/vulkan-amdgpu-pro_${MY_PV_FULL}_i386.deb -> vulkan-amdgpu-pro_${MY_PV}_i386.deb )
"  
```

Also remember the tab before abi_x86_64 and abi_x86_32

Reply:
Noted thanks!

Reply:
Done!